enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/MultiRealmAuthManager.java

```diff
@@ -101,10 +101,26 @@ public EnterpriseSecurityContext login( Map<String,Object> authToken ) throws In
         {
             securityContext = new StandardEnterpriseSecurityContext(
                     this, (ShiroSubject) securityManager.login( null, token ) );
-            if ( logSuccessfulLogin )
+            AuthenticationResult authenticationResult = securityContext.subject().getAuthenticationResult();
+            if ( authenticationResult == AuthenticationResult.SUCCESS )
             {
-                securityLog.info( securityContext, "logged in" );
+                if ( logSuccessfulLogin )
+                {
+                    securityLog.info( securityContext, "logged in" );
+                }
+            }
+            else if ( authenticationResult == AuthenticationResult.PASSWORD_CHANGE_REQUIRED )
+            {
+                securityLog.info( securityContext, "logged in (password change required)" );
             }
+            else
+            {
+                String errorMessage = ((StandardEnterpriseSecurityContext.NeoShiroSubject) securityContext.subject())
+                        .getAuthenticationFailureMessage();
+                securityLog.error( "[%s]: failed to log in: %s", escape( token.getPrincipal().toString() ), errorMessage );
```

Comment:
could use `securityLog.error( securityContext, "failed to log in: %s", errorMessage );`